### DISCUSSION TITLE
Exsiting HA Motion sensor/switch to trigger Paradox Alarm Zone

### DISCUSSION THREAD

**Discussion by burghs** [ParadoxAlarmInterface/pai#363] [Q&A]
hi there, 

## Alarm system
I have my Spectra SP6000 with IP150 integrated with Home Assistant

## Environment
My Home Assistant is built on a Intel NUC and I am running the MQTT broker locally.

## Question
The integration works great, I would just like to find out if it is possible to publish back to the panel using the PAI MQTT topics at all or is there perhaps a setting to enable this? 

The reason for asking is that I have my panic alarm set to a zone on the SP6000 alarm panel and it is seen as a "motion" entity through the PAI integration with HA. If I manually attempt to set the payload for the panic alarm "paradox/states/zones/panicalarm/open" to true, it updates state of the entity to "Detected" but it doesn't actually trigger on the alarm panel itself. 

I have seen some threads here regarding the panic alarm itself but it doesn't 100% cover the part in interacting with the entity from HA side. Any guidance or links would be appreciated.

**Comment by yozik04** [comment]
You can control a PGM through PAI. That PGM can be physically wired to a zone. This way you can trigger an alarm on that zone. You can find more details in the issues.
